src/sensors/icm20948sensor.cpp

```diff
@@ -236,6 +241,16 @@ void ICM20948Sensor::motionSetup() {
         }
     }
 
+    if(imu.setDMPODRrate(DMP_ODR_Reg_Accel, 1.25) == ICM_20948_Stat_Ok)
```

Comment:
missing this-> here

Reply:
It would be pretty out of place right there given the context around it.